docs/user/dashboard.asciidoc

```diff
@@ -34,54 +32,60 @@ dashboards are not visible. For more information, see <<xpack-security-authoriza
 [role="screenshot"]
 image::images/dashboard-read-only-badge.png[Example of Dashboard's read only access indicator in Kibana's header]
 
-[float]
-[[dashboard-getting-started]]
-=== Interact with dashboards
-
-When you open *Dashboard*, you're presented an overview of your dashboards. 
-If you don't have any dashboards, you can add 
-<<add-sample-data, sample data sets>>,
-which include pre-built dashboards. 
-
-Once you open a dashboard, you can filter the data
-by entering a search query, changing the time filter, or clicking 
-in the visualizations, searches, and maps. If a dashboard element has a stored query, 
-both queries are applied.
-
 --
 
 [[dashboard-create-new-dashboard]]
 == Create a dashboard
 
-To create a dashboard, you must have data indexed into {es}, an index pattern 
-to retrieve the data from {es}, and 
-visualizations, saved searches, or maps. If these don't exist, you're prompted to 
-add them as you create the dashboard.
+To create a dashboard, you must have data indexed into {es}, an index pattern
+to retrieve the data from {es}, and
+visualizations, saved searches, or maps. If these don't exist, you're prompted to
+add them as you create the dashboard, or you can add
+<<add-sample-data, sample data sets>>,
+which include pre-built dashboards.
 
 For an end-to-end example, see <<tutorial-build-dashboard, Building your own dashboard>>.
 
-. Open *Dashboard.*
-. Click *Create new dashboard.*
-. Click *Add*.
-. Use *Add panels* to add elements to the dashboard.
-+
+To begin, open *Dashboard*, then click *Create new dashboard.*
+
+[float]
+[[dashboard-add-elements]]
+=== Add elements
+
 The visualizations, saved searches, and maps
-are stored in panels that you can move and resize. A
-menu in the upper right of the panel has options for customizing
-the panel. You can add elements from 
+are stored as elements in panels that you can move and resize. The
+menu in the upper right of the panel includes customization options. You can add elements from
 multiple indices, and the same element can appear in multiple dashboards.
+
+To create an element:
+
+. Click *Create new*.
+. On the *New Visualization* window, click the visualization type.
 +
 [role="screenshot"]
-image:images/Dashboard_add_visualization.png[Example add visualization to dashboard]
+image:images/Dashboard_add_new_visualization.png[Example add new visualization to dashboard]
++
+For information on how to create visualizations, see <<visualize,Visualize>>.
++
+For information on how to create Maps, see <<maps,Elastic Maps>>.
+
+To add an existing element:
 
-. When you're finished adding and arranging the panels,
-*Save* the dashboard.
+. Click *Add*.
+
+. On the *Add panels* flyout, select the panel.
++
+[role="screenshot"]
+image:images/Dashboard_add_visualization.png[Example add visualization to dashboard]
++
+When a dashboard element has a stored query,
```

Comment:
Maybe move this sentence up, indented under the step, so that it doesn't get lost under the image?